src/vscode-dts/vscode.d.ts

```diff
@@ -7055,6 +7055,23 @@ declare module 'vscode' {
 		 */
 		clear(): void;
 
+		/**
+		 * Called when a custom editor is opened.
+		 * @param document - The text document that the editor is being used for.
+		 * @param webviewPanel - The webview panel in which the custom editor's UI is displayed.
+		 *                        Use the `webviewPanel.webview.html` to set the HTML content.
+		 * @param token - A cancellation token indicating that the editor opening is being canceled.
+		 *
+		 * Example usage:
+		 * ```typescript
+		 * resolveCustomEditor(document, webviewPanel, token) {
+		 *   webviewPanel.webview.html = '<html><body><h1>Hello, Custom Editor!</h1></body></html>';
+		 *   return Promise.resolve();
+		 * }
+		 * ```
+		 */
+		resolveCustomEditor(document: TextDocument, webviewPanel: WebviewPanel, token: CancellationToken): Thenable<void>;
```

Comment:
Having `resolveCustomEditor` in the `OutputChannel` is wrong. This method is defined and documented here

https://github.com/microsoft/vscode/blob/1dfb5777c9bfa361a447283f941c211bdab3fd05/src/vscode-dts/vscode.d.ts#L10162

Reply:
I made a new push, just inserting the small example, should be okay now?